src/AppInstallerCLICore/ExecutionProgress.cpp

```diff
@@ -192,7 +192,6 @@ namespace AppInstaller::CLI::Execution
         for (size_t i = 0; !m_canceled; ++i)
         {
             constexpr size_t repetitionCount = 20;
-            ApplyStyle(i % repetitionCount, repetitionCount, true);

```

Comment:
>ApplyStyle(i % repetitionCount, repetitionCount, true); [](start = 12, length = 55)

Why has this been removed?  Seems unrelated.

Reply:
When I run locally to verify the UI behavior, the spinner applies the theme color, but on exit, it's not reverting to default. And this method is called NoVT, so I thought we left it here by accident?

---
In reply to: [453968675](https://github.com/microsoft/winget-cli/pull/493#discussion_r453968675) [](ancestors = 453968675)

Reply:
No style is applied in the NoVT case.  If you are seeing that happening, it is due to a different bug.

---
In reply to: [453998322](https://github.com/Microsoft/pkg/pull/493#discussion_r453998322) [](ancestors = 453998322,453968675)